rviz2/src/main.cpp

```diff
@@ -40,7 +41,15 @@
 
 int main(int argc, char ** argv)
 {
-  QApplication qapp(argc, argv);
+  // remove ROS arguments before passing to QApplication
+  std::vector<std::string> non_ros_args = rclcpp::remove_ros_arguments(argc, argv);
+  std::vector<char *> non_ros_args_c_strings;
```

Comment:
Do these `char *`'s need to be non-`const`?

Reply:
Yes, I think so. I believe the compiler was complaining about discarding const-ness when passing to QApplication.